lib/DDG/Spice/FirstRobotics/TeamInfo.pm

```diff
@@ -0,0 +1,46 @@
+package DDG::Spice::FirstRobotics::TeamInfo;
+
+# ABSTRACT: Get info on teams in the FIRST Robotics Competition
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://www.thebluealliance.com/api/v2/team/frc$1?X-TBA-App-Id=duckduckgo:search-engine:1';
+
+# Obtain the years a team has competed
+spice alt_to => {
+    years_competed => {
+        is_cached => 1,
+        proxy_cache_valid => "200 1d",
+        to => 'https://www.thebluealliance.com/api/v2/team/frc$1/years_participated?X-TBA-App-Id=duckduckgo:search-engine:1'
+    }
+};
+
+triggers any => 'frc', 'frc team', 'first robotics team', 'first robotics competition';
```

Comment:
Hmm handling queries like that is a little touchy because we're not validating that "based" is something we know we can provide a correct answer for.

For example queries like "which is the rival team of frc 811?" or "where is frc 811 playing next" would be triggered and return a result that would be irrelevant to the query.

e.g. https://beta.duckduckgo.com/?q=rival+team+of+frc+1111&ia=reference

Actually, that could still happen with a `startend` trigger IA because we're not considering the rest of the query: 

e.g. https://beta.duckduckgo.com/?q=frc+team+885+rivals&ia=reference